csrc/ops/composite.cpp

```diff
@@ -534,6 +534,20 @@ SdpfaFwdResult sdpfa_fwd(
   return {output, log_sumexp, philox_seed, philox_offset};
 }
 
+namespace {
+template <typename T>
+void checkAllEqual(std::initializer_list<T> elements) {
```

Comment:
Nice template, we can propagate this to `sdpfa_fwd` above as well (does not have to be this PR)

Reply:
Done!